packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -758,6 +787,155 @@ module.exports = (
           });
         }
 
+        return {};
+      },
+    },
+    {
+      method: 'POST',
+      path: '/recovery_email/secondary/resend_code',
+      options: {
+        auth: {
+          strategy: 'sessionToken',
+        },
+        validate: {
+          payload: {
+            email: validators.email().required(),
+          },
+        },
+        response: {},
+      },
+      handler: async function(request) {
+        log.begin('Account.RecoveryEmailSecondaryResend', request);
+
+        const sessionToken = request.auth.credentials;
+        const ip = request.app.clientAddress;
+        const geoData = request.app.geo;
+        const { email } = request.payload;
+
+        await customs.check(
+          request,
+          sessionToken.email,
+          'recoveryEmailSecondaryResendCode'
```

Comment:
Are new customs checks hooked up automatically?

Reply:
Nope! I'll update this